### ISSUE TITLE
Would be useful to have comments be emitted in the *.protobuf file

### ISSUE THREAD

**Issue by KatamariJr** [anjmao/go2proto#4]
Just a suggestion

**Comment by anjmao** [comment]
@KatamariJr I agree, but user need to be able to disable this feature with a flag. I was using go2proto when I needed to migrate from swagger to grpc and swagger generated comments was useless.
